.changeset/polite-eyes-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/react-ui": patch
+---
+
+Update alert component
```

Comment:
Please add a bit more info here regarding what was updated, also there is a breaking change with the `renderContent` prop typing changing so it's not retro-compatible, you might want to include that and use a `minor` bump. (Ideally it would be a `major` but for now we're not ready for 1.0.0)

Reply:
I have updated the comment. Please let me know if you want to add some specific text.